recipes/ruy/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,161 @@
+/* Copyright 2019 Google LLC. All Rights Reserved.
```

Comment:
can't we have a simple test package without a google copyright?

Reply:
TBH I don't know how this library works, I need tensorflow-lite and this was a requirement.
So I copy-pasted one of their examples.

Reply:
Yes but a file with Apache 2.0 license in CCI is really annonying. Now if I want to change this file, I won't even be allowed to remove this copyright but I'll need to explain my modifications in this file...

Reply:
I see. I'll simplify the test package tomorrow. I'll just call some random function and include some of the headers. That should be enough for the testing we do here.